src/components/playerOverview/PlayerDetails.tsx

```diff
@@ -59,6 +59,15 @@ export const PlayerDetails = React.memo(function PlayerDetails() {
                 <Flex direction="column" padding="16px">
                     <Text>{`Total Number of Games: ${numberOfMatches}`}</Text>
                     <Text>{`Winrate: ${playerWinRate}%`}</Text>
+                    <Text>
+                        Favorite commander:{" "}
```

Comment:
remove this space character

side note: while this doesn't cause problems on web, on react native, this will actually be an error and throw an exception :)

Reply:
If you are intending for the space to be a separator before the link, you can make the string "favorite commander: "

however, if that's the case, I recommend that we simply make the label and value appear on 2 separate lines (kinda foreshadowing the replacement of the link with an image anyways)

Reply:
Moved to two lines